src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -193,14 +193,19 @@ private void MoveProjectJsonArtifactsToBackup(MigrationReport migrationReport)
 
         private void BackupProjects(MigrationReport migrationReport)
         {
+            var projectDirectories = new List<DirectoryInfo>();
             foreach (var report in migrationReport.ProjectMigrationReports)
             {
-                var backupPlan = new MigrationBackupPlan(
-                    new DirectoryInfo(report.ProjectDirectory),
-                    _workspaceDirectory);
-
-                backupPlan.PerformBackup();
+                projectDirectories.Add(new DirectoryInfo(report.ProjectDirectory));
             }
+
+            var backupPlan = new MigrationBackupPlan(
+                projectDirectories,
+                _workspaceDirectory);
+
+            backupPlan.PerformBackup();
+
+            Reporter.Output.WriteLine($"Files backed up to {backupPlan.RootBackupDirectory.FullName}");
```

Comment:
@livarcocc: Here's another string we'll want to extract for localization. I didn't do this one yet only because I wanted to avoid merge conflicts with you.